catalog/transformer/catalog-transformer-rtf/src/main/java/ddf/catalog/transformer/output/rtf/RtfTemplate.java

```diff
@@ -102,11 +104,17 @@ private RtfRow appendProperty(
               .map(this::imageFromStream)
               .orElse(text(EMPTY_VALUE));
 
-      return row(entry.getKey(), picture)
-          .topCellBorder()
-          .rightCellBorder()
-          .leftCellBorder()
-          .bottomCellBorder();
+      try {
+        return row(entry.getKey(), picture)
+            .topCellBorder()
+            .rightCellBorder()
+            .leftCellBorder()
+            .bottomCellBorder();
+      } catch (RtfException e) {
+        // Failed likely due to the fact this thumbnail is not a JPG or PNG
```

Comment:
Could be worth logging the exception here.

Reply:
I also considered mkaing the entry say something like "GIF not compatible with RTF". So that way it wasn't purely filtered out, but the user would be aware why it wasn't listed.